src/widget/wnumberrate.cpp

```diff
@@ -32,11 +31,14 @@ void WNumberRate::setup(const QDomNode& node, const SkinContext& context) {
 void WNumberRate::setValue(double dValue) {
     double vsign = dValue - 1;
 
-    char sign = '+';
-    if (vsign < -0.00000001) {
+    QString sign(' ');
```

Comment:
QChar ?

Reply:
It seems that QChar does not help and QString is the right one when appending